test/.eslintrc

```diff
@@ -5,7 +5,10 @@
   globals:
     expect: false
     sinon: false
+    $: false
+    browser: false
 
   rules:
     no-invalid-this: 0
     no-unused-expressions: 0
+    vars-on-top: 0
```

Comment:
Why negate this rule?

Reply:
We don't have to negate it, I'll fix up the tests to not trigger it.